pkg/asset/machines/openstack/machines.go

```diff
@@ -91,7 +91,7 @@ func provider(clusterID string, platform *openstack.Platform, mpool *openstack.M
 			Size:       pointer.Int64Ptr(int64(mpool.Size)),
 		},*/
 		Image:          osImage,
-		CloudName:      CloudName,
```

Comment:
Looks like the CloudName const is now unused. We should also remove it if that's effectively the case.

https://github.com/openshift/installer/pull/1858/files#diff-a2bd450b1cd3199485e614928c597a91R27

Reply:
no, it's still used... I actually removed the definition at beginning and it lead to some problem...

Reply:
I will try another time and paste result here

Reply:
here's the error:

+ go build -ldflags ' -X github.com/openshift/installer/pkg/version.Raw=unreleased-master-1126-gd77481175181d604f4b6e41d7b26d7655eec35be-dirty -X github.com/openshift/installer/pkg/version.Commit=d77481175181d604f4b6e41d7b26d7655eec35be -s -w' -tags ' release' -o bin/openshift-install ./cmd/openshift-install
# github.com/openshift/installer/pkg/asset/manifests
pkg/asset/manifests/cloudproviderconfig.go:92:4: undefined: "github.com/openshift/installer/pkg/asset/machines/openstack".CloudName
pkg/asset/manifests/openshift.go:108:4: undefined: "github.com/openshift/installer/pkg/asset/machines/openstack".CloudName